### ISSUE TITLE
computer.sleep is not defined

### ISSUE THREAD

**Issue by hasezoey** [mrmakeit/OCJS#48]
(in the EEPROM's Eval)
```js
function onSignal() {
    computer.sleep(0);
    term.write('GOT SIGNAL ' + JSON.stringify(arguments));
}
```
but other `computer.` functions work

**Comment by mrmakeit** [comment]
I think the latest commit in the 1.10 branch has the working sleep implementation.

**Comment by hasezoey** [comment]
i have the latest version of the branch 1.12.2 (because this is the only i use~~, sorry~~)

does this mean it is not in the 1.12.2?

**Comment by hasezoey** [comment]
yeah i see, made the 1.10.2 version, and getting "empty" onSignals (or ticks) and yes, sleep is working
todo: make it work in 1.12.2, thanks
